docs/using/installing/index.rst

```diff
@@ -510,6 +510,11 @@ The file must always include ``version`` and ``control-service`` items similar t
       "hostname": "${CONTROL_NODE}"
       "port": 4524
 
+The value of the hostname field should be a hostname or IP that is routable from all your node agents.
+
+When configuring node agents, consider whether the control service location you choose will have multiple possible addresses, and ensure the hostname you provide is the correct one.
+You should never choose ``127.0.0.1`` or ``localhost`` as the hostname, even if the control service is on same machine as the node agent.
+
```

Comment:
Still missing one piece of information: the interface you choose will be the one that linked traffic will be routed over. So if you're in environment where some interfaces have bandwidth costs and some are free (e.g. AWS) make sure you choose the private one where bandwidth costs don't apply.